zellij-server/src/unit/snapshots/zellij_server__screen__screen_tests__send_cli_query_tab_names_action.snap

```diff
@@ -0,0 +1,30 @@
+---
```

Comment:
test case added! i've no idea if this is ok @imsnif

Reply:
Hey, good start with this. I adjusted the test so that it tests for the `Log` action that we added rather than the `Render` (because that won't really tell us anything...)

I also merged from main and made some minor naming changes.

Thanks for your work on this! I'll merge once the CI is green.

Reply:
thanks for your explanation, happy to learn that all